tests/neptune/new/test_client.py

```diff
@@ -269,14 +270,16 @@ def test_debug_mode(self):
         self.assertNotIn(str(project._id), os.listdir(".neptune"))
 
     def test_offline_mode(self):
-        project = init_project(name=self.PROJECT_NAME, mode="offline")
-        project["some/variable"] = 13
-        with self.assertRaises(NeptuneOfflineModeFetchException):
-            project["some/variable"].fetch()
-        self.assertIn(str(project._id), os.listdir(".neptune/offline"))
-        self.assertIn(
-            "data-1.log", os.listdir(".neptune/offline/{}".format(project._id))
-        )
+        with self.assertRaises(NeptuneException):
+            init_project(name=self.PROJECT_NAME, mode="offline")
+        # project = init_project(name=self.PROJECT_NAME, mode="offline")
```

Comment:
?

Reply:
We've agreed with product that we're not allowing to invoke `init_project` in `offline` mode, we'll improve it when implementing ModelRegistry.

Reply:
I know but please don't leave commented out code. I think black/pylint should warn about it.

Reply:
But I think that we'll support this soon (several weeks from now), so we may easily uncomment it.
But if you're strongly against I can drop it. Your call ;)

Reply:
Drop it. Things may change, memory is volatile :wink: